features/step_definitions/form_steps.rb

```diff
@@ -52,6 +52,10 @@
 # Then "there {is} a field {string}" do |present, field|
 # end
 
+Then "field {string} has value {string}" do |field, value|
+  assert_equal value, find_field(field)[:value]
+end
+
```

Comment:
I'm surprised a step like this was not already defined or even part of Cucumbers gem itself.